public/app/features/dashboard-scene/pages/DashboardScenePageStateManager.ts

```diff
@@ -50,14 +55,28 @@ export class DashboardScenePageStateManager extends StateManagerBase<DashboardSc
   // This is a simplistic, short-term cache for DashboardDTOs to avoid fetching the same dashboard multiple times across a short time span.
   private dashboardCache?: DashboardCacheEntry;
 
+  constructor(state: DashboardScenePageState) {
+    super(state);
+
+    appEvents.subscribe(ReloadDashboardEvent, ({ payload }) => this.reloadDashboard(payload));
```

Comment:
Is there a way to not rely on a global event? Dashboards can be embedded (used as a component on another page)

Reply:
Removed the event all together. We can rely on `getDashboardScenePageStateManager()` to actually retrieve the page manager.

The problem was that the state manager was not part of the scene graph. Creating a separate event bus instance just for this purpose sounds a little bit extreme given the current approach.

WDYT?

Reply:
looks ok